tests/testthat/_snaps/table_egt03.md

```diff
@@ -3,102 +3,102 @@
     Code
       res
     Output
-                                    LOW     NORMAL      HIGH   Missing 
-      —————————————————————————————————————————————————————————————————
-      Placebo (N=86)                                                   
-        n                            0        333        0        3    
-        LOW                          0     84 (25%)      0        0    
-        NORMAL                       0    249 (74.1%)    0        0    
-        HIGH                         0         0         0        0    
-        Missing                      0         0         0     3 (0.9%)
-      Xanomeline High Dose (N=84)                                      
-        n                            0        327        0        1    
-        LOW                          0     82 (25%)      0        0    
-        NORMAL                       0    245 (74.7%)    0        0    
-        HIGH                         0         0         0        0    
-        Missing                      0         0         0     1 (0.3%)
-      Xanomeline Low Dose (N=84)                                       
-        n                            0        335        0        1    
-        LOW                          0     84 (25%)      0        0    
-        NORMAL                       0    251 (74.7%)    0        0    
-        HIGH                         0         0         0        0    
-        Missing                      0         0         0     1 (0.3%)
+                                       LOW         NORMAL      HIGH   Missing 
+      ————————————————————————————————————————————————————————————————————————
+      Placebo (N=86)                                                          
+        n                               30           303        0        3    
```

Comment:
hi @edelarua , could you check this again, 86 should be the sum of 30, 303, 0 and 3. 

could you check the normal one? it is too high? @khatril

Reply:
@shajoezhu I have added the `DTYPE == "AVERAGE"` filter to egt03 as well so these numbers should be updated now.